configure

```diff
@@ -29,7 +29,7 @@ check_toolchain()
     local clang_version
 
     : ${PKG_CONFIG:=pkg-config}
-    : ${CC=gcc}
+    : ${CC:=gcc}
```

Comment:
The inconsistency between `:=` and `=` is certainly a bit weird, but setting CC already works AFAICT?

```sh
CC=clang ./configure ; grep CC: config.mk     
CC:=clang
```

Reply:
Yes, you are right, this isn't needed.  I've dropped this patch.